library/src/main/res/values/strings.xml

```diff
@@ -46,7 +46,7 @@
     <string name="chucker_tab_errors">Errors</string>
     <string name="chucker_notification_category">Chucker notifications</string>
     <string name="chucker_share_error_title">Error report</string>
-    <string name="chucker_share_error_content"><![CDATA[Date: %1$s\nException: %2$s\nTag: %3$s\nMessage: %4$s\n\n%5$s]]></string>
+    <string name="chucker_share_error_content"><![CDATA[Date: %s\nException: %s\nTag: %s\nMessage: %s\n\n%s]]></string>
```

Comment:
Apparently this should not be in the diff.
Can you please remove it?

Reply:
Sure. Done.